src/elements/block/core-embed/embed/iframe/iframe.spec.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import { render } from '@testing-library/react'
+import { Iframe } from '.'
+
+it('should render the embeded video', () => {
+  const { getByTestId } = render(
+    <Iframe
+      attributes={{
+        url: 'https://www.youtube.com/embed/UGFCbmvk0vo',
+        providerNameSlug: 'embed-handler'
+      }}
+    />
+  )
+  expect(getByTestId('embed-handler').getAttribute('src')).toEqual(
+    'https://www.youtube.com/embed/UGFCbmvk0vo'
+  )
+})
```

Comment:
Can you move all your tests into describe blocks like so? This means in the test interface the component name is displayed on failure.

```suggestion
describe('Iframe', () => {
  it('should render the embeded video', () => {
    const { getByTestId } = render(
      <Iframe
        attributes={{
          url: 'https://www.youtube.com/embed/UGFCbmvk0vo',
          providerNameSlug: 'embed-handler'
        }}
      />
    )
    expect(getByTestId('embed-handler').getAttribute('src')).toEqual(
      'https://www.youtube.com/embed/UGFCbmvk0vo'
    )
  })
})
```